pkg/registry/core/service/portallocator/controller/repair.go

```diff
@@ -139,7 +140,7 @@ func (c *Repair) runOnce() error {
 			case nil:
 				if stored.Has(port) {
 					// remove it from the old set, so we can find leaks
-					stored.Release(port)
+					_ = stored.Release(port)
```

Comment:
This is unnecessary

Reply:
oh this is modified to make goland happy. restored.